lib/source_finder/source_file_globber.rb

```diff
@@ -30,11 +30,27 @@ def extra_ruby_files_arr
       @extra_ruby_files_arr ||= %w(Rakefile)
     end
 
+    def default_source_files_exclude_glob
+      '**/vendor/**'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.